dotcom-rendering/src/lib/contributions.ts

```diff
@@ -160,7 +160,7 @@ export const hasCmpConsentForArticleCount = (): Promise<boolean> => {
 			resolve(true);
 		}
 		onConsentChange(({ ccpa, tcfv2, aus }) => {
-			if (ccpa || aus) {
+			if (ccpa ?? aus) {
```

Comment:
While this works, I think the intent would be clearer with coercing these to `boolean`, as otherwise we rely on the [falsy evaluation of `undefined` by `if` operator](https://developer.mozilla.org/en-US/docs/Glossary/Falsy#examples).

```suggestion
			if (!!ccpa || !!aus) {
```

Reply:
is that different from a falsy evaluation during coercion to a boolean?